README.md

```diff
@@ -35,18 +35,18 @@ As a strawman, we propose **the 99th percentile (P99)** as a lowest common denom
 
 ## The 'Benchmark Effect'
 
-As explained in the introduction, defining a delay metric is not just about choosing a percentile. The layer to measure and the background traffic pattern to use also have to be defined. As soon as these have been settled on, researchers, product engineers, etc. tend to optimize around this set of conditions---the so-called 'benchmark effect'. It is possible that harmonizing around one choice of percentile will lead to a benchmark effect. However, a percentile metric seems robust against such perverse incentives, because it seems hard to contrive performance results that fall off a cliff just beyond a certain percentile. Nonetheless, even if there were a benchmark effect, it would be harmless if the percentile chosen for the benchmark realistically reflected the needs of most applications. {ToDo: better wording for last sentence.}
+As explained in the introduction, defining a delay metric is not just about choosing a percentile. The layer to measure and the background traffic pattern to use also have to be defined. As soon as these have been settled on, researchers, product engineers, etc. tend to optimize around this set of conditions---the so-called 'benchmark effect'. It is possible that harmonizing around one choice of percentile will lead to a benchmark effect. However, a percentile metric seems robust against such perverse incentives, because it seems hard to contrive performance results that fall off a cliff just beyond a certain percentile. [This is a little weak.  An intermittent outtage on a link that causes packets to queue up would cause delay spikes that might only show up in percentiles greater than P99.  These outtages would be ignored if P99 were the metric.] Nonetheless, even if there were a benchmark effect, it would be harmless if the percentile chosen for the benchmark realistically reflected the needs of most applications. {ToDo: better wording for last sentence.}
```

Comment:
Whether low likelihood effects might be concealed by choosing P99 as the metric is surely a different question from whether people can *contrive* to conceal poor performance when they have to report against a benchmark test.

Nonetheless, I agree that this argument is weak. Any ideas on improving it? I could delete the section without harming the paper.

Reply:
I guess the contrivance would involve ensuring that the glitch affected less than 1% of packets, with no concern for how bad those <1% were affected.  e.g. having a timer that reboots the device every so often to avoid the affects of a memory leak showing up in the P99 score. 

I wonder whether we need a strong argument that P99 is more robust against the benchmark effect than other metrics would be.  It doesn't seem to me that it is any more or any less robust.  The final point you make is the more important one.   To date, the de facto single metric that gets quoted is average latency, which we argue is pretty worthless in predicting QoE.  So the industry has been suffering terribly with the benchmark effect of using average latency for years (and people ignored bufferbloat, etc.).